package.json

```diff
@@ -47,6 +47,7 @@
     "chai": "^3.5.0",
     "clamp": "^1.0.1",
     "classnames": "^2.2.5",
+    "copy-to-clipboard": "^3.0.5",
```

Comment:
This is pretty gnarly to do on your own. This module creates a span on the fly, select the contents, and then executes the copy command, and finally cleans up after itself.